src/dma/collector/sql/sources/postgres/collection-table_details.sql

```diff
@@ -435,3 +435,35 @@ select :PKEY as pkey,
   COALESCE(src.toast_index_read, 0) as toast_index_read,
   current_database() as database_name
 from src;
+
+
+-- name: collection-postgres-tables-with-no-primary-key
+with src as (
```

Comment:
As a note fure future enhancements (not in this PR), we should add this as a CTE to the `table-details` query above.  We could then add a `has_primary_key` indicator column to the extract.  This will let us leverage the existing extract against the table details.